internal/pkg/cli/svc_delete.go

```diff
@@ -99,26 +99,36 @@ func newDeleteSvcOpts(vars deleteSvcVars) (*deleteSvcOpts, error) {
 
 // Validate returns an error if the user inputs are invalid.
```

Comment:
nit: can we update the `Validate` comment to mention that it's for optional inputs.